arviz/utils.py

```diff
@@ -696,5 +696,6 @@ class HtmlTemplate:
                   </div>
             </li>
             """
+    _, css_style = xr.core.formatting_html._load_static_files()  # pylint: disable=protected-access
```

Comment:
I am fine with enforcing the lazy load but we then have to update requirements to xarray 0.16.1 too, otherwise we need and if/else.

Reply:
True, what should we do? Have our own css file, or copy the lazy load function?

Reply:
I gravitate towards requiring xarray 0.16.1 so this code works without problem or having our lazy load function that calls xarray's one if possible and gets css style otherwise.

Reply:
Let's update xarray to 0.16.1

Reply:
Just to warn you, this line is definitely using an internal xarray function that could go away or be disabled in the future. That's why you had to add the lint comment to disable protected-access :).

If you are OK copying the CSS file into arViz, that would be my preference for the safest/simplest way to do this (without any chance of things breaking in the future).

Otherwise, if for some reason you need to keep it in sync with xarray's version, we could definitely consider adding a public API option for getting this information from xarray. In that case, please open an issue in xarray's issue tracker.

Reply:
Yes, we are using internal function. Until we figure out a good way to solve this, I think we are ok with this solution.

xarray should not (and probably are not) worry changing their internal functionality.